modules/jupyter/variables.tf

```diff
@@ -23,9 +23,10 @@ variable "default_backend_service" {
 }
 
 variable "members_allowlist" {
-  type        = string
-  default     = ""
+  type    = list(string)
```

Comment:
Do we still need memebrs_allowlist if we are removing `google_iap_web_backend_service_iam_binding`?